common/src/main/java/com/datastax/oss/common/sink/config/CassandraSinkConfig.java

```diff
@@ -352,27 +368,27 @@ private void deprecatedSecureBundle(Map<String, String> connectorSettings) {
         Function.identity());
   }
 
-  static void decodeBase64EncodedSecureBundle(Map<String, String> javaDriverSettings) {
+  static void decodeBase64EncodedFile(String key, Map<String, String> javaDriverSettings) {
     // if the path is base64:xxxx we decode the payload and create
     // a temporary file
     // we are setting permissions such that only current user can access the file
     // the file will be deleted at JVM exit
-    String encoded = javaDriverSettings.get(SECURE_CONNECT_BUNDLE_DRIVER_SETTING);
+    String encoded = javaDriverSettings.get(key);
     if (encoded != null && encoded.startsWith("base64:")) {
       try {
         encoded = encoded.replace("\n", "").replace("\r", "").trim();
         encoded = encoded.substring("base64:".length());
         byte[] decoded = Base64.getDecoder().decode(encoded);
-        Path file = Files.createTempFile("cassandra.sink.securebundle", ".zip");
+        Path file = Files.createTempFile("cassandra.sink.", ".tmp");
         Files.setPosixFilePermissions(file, PosixFilePermissions.fromString("rw-------"));
         file.toFile().deleteOnExit();
         Files.write(file, decoded);
         String path = file.toAbsolutePath().toString();
-        log.info("Decoded bundle to temporary file {}", path);
-        javaDriverSettings.put(SECURE_CONNECT_BUNDLE_DRIVER_SETTING, path);
+        log.info("Decoded {} to temporary file {}", key, path);
```

Comment:
shouldn't this be `debug` level?

Reply:
I left it at `info`, because this way you can see where is the temporary directory that we are using.

do you think we should use debug ?
in Pulsar IO it is basically NOT POSSIBLE to turn on "debug" level currently, it logs only "info" and higher

Reply:
such information may be not useful for end-user, that is why I proposed debug. If it is not feasible in pulsar, let's leave the info. It should not produce a lot of logs because it is executed only once when the app is starting.